tests/k8s/multi-node/Vagrantfile

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.require_version ">= 1.8.3"
+
+$build_docker_image = <<SCRIPT
```

Comment:
It would be nice to have some comments before each script section outlining what they do.